app/src/main/res/values/strings.xml

```diff
@@ -218,6 +224,14 @@
     <string name="event_time">Event Time</string>
     <string name="go_back_to_map_fragment">go back to map fragment</string>
 
+    <!-- Notifications -->
+    <string name="event_notification_channel_id">event_channel</string>
+    <string name="event_notification_channel_name">Event Channel</string>
+    <string name="event_notification_channel_description">Event reminder</string>
+    <string name="event_start_soon">Event %s will start soon!</string>
```

Comment:
It could also be an option to display "Event %s will start in %d minutes!"

Reply:
We could. But since we're only notifying 15 minutes before, it's kind of fixed. Also, we don't want the time left to be displayed like "Event eventName will start in 15 min" because it can be misleading, if time has passed since the notification popped up.